_i18n/de/resources/moneropedia/transaction.md

```diff
@@ -1,22 +1,23 @@
 ---
-terms: ["transaction", "transactions"]
-summary: "a cryptographically signed container that details the transfer of Monero to a recipient (or recipients)"
+terms: ["transaction", "transactions", "Transaktion", "Transaktionen"]
+summary: "Ein kryptografisch signierter Behälter, der einen Monero-Transfer einem (oder mehreren) Empfänger(n) zuweist"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A cryptographically signed container that details the transfer of Monero to a recipient (or recipients).
+Ein kryptografisch signierter Behälter, der einen Monero-Transfer einem (oder mehreren) Empfänger(n) zuweist.
 
-The parameters of a transaction contain one or more recipient addresses with corresponding amounts of funds and a @ring-size parameter that specifies the number outputs bound to the transaction. The more outputs that are used, a higher degree of obfuscation is possible, but that comes with a cost. Since a transaction gets larger with more outputs, the transaction fee will be higher.
+Die Parameter einer Transaktion enthalten eine oder mehrere Empfängeradressen mit entsprechenden Geldbeträgen und einen Parameter bezüglich der @Ringgröße, welcher die Anzahl der an die Transaktion gebundenen Outputs festlegt. Je mehr Outputs verwendet werden, desto höher kann der Grad an Verschleierung sein. Dies hat jedoch seinen Preis: Da eine Transaktion durch mehr Outputs größer wird, werden die Transaktionsgebühren höher ausfallen.
```

Comment:
translation is fine, but @erciccione please verify if the content is right... isn't ring size fixed to 11 since a while?

Reply:
> isn't ring size fixed to 11 since a while?

it is, yes. This entry will need to be updated.